internal/compiler/generator/cpp.rs

```diff
@@ -2821,6 +2821,12 @@ fn compile_expression(expr: &llr::Expression, ctx: &EvaluationContext) -> String
             "slint::cbindgen_private::EasingCurve(slint::cbindgen_private::EasingCurve::Tag::CubicBezier, {}, {}, {}, {})",
             a, b, c, d
         ),
+        Expression::EasingCurve(EasingCurve::EaseInElastic) => "slint::cbindgen_private::EasingCurve()".into(),
```

Comment:
you will have to pass something like` slint::cbindgen_private::EasingCurve::Tag::EaseInElastic` in the constructor.

Reply:
Let me know if that looks good now.